src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/PropertyBasedEmbeddingProviderConfig.java

```diff
@@ -10,23 +10,11 @@
 import java.util.Optional;
 import org.eclipse.microprofile.config.spi.Converter;
 
-@ConfigMapping(prefix = "jsonapi.embedding.provider")
+@ConfigMapping(prefix = "jsonapi.embedding")
```

Comment:
Not done as part of this ticket but why we dropped preceeding `stargate` to the property. All other configuration has `stargate` in them.

Reply:
Initially, I changed it for testing purposes. Added back